### DISCUSSION TITLE
Noobie question, what do I input in RTMP server and keyfield for Twitch/YouTube?

### DISCUSSION THREAD

**Discussion by atlas7ws** [sorayuki/obs-multi-rtmp#173] [Q&A]
Sorry for noob question. If I want to stream to both twitch and youtube at the same time, what do I input in the RTMP server and key field? 

I think I know what to put in the key field, but not sure on the server part. Can anyone advise? Sorry I'm a noob.

**Comment by 3liza-git** [comment]
I'm having the same problem. Where's the documentation?

**Comment by mdamaged** [comment]
https://help.livestream.com/hc/en-us/articles/12653224769937-How-Do-I-Find-RTMP-Stream-Settings-For-Popular-Providers-
